### PR TITLE
add sqlite db

### PR DESCRIPTION
**Pull request by mutahirqureshi**

### CODE REVIEW

File: fake_db.py [hunk 1]
```diff
@@ -9,17 +9,14 @@
     }
 """
 
-class FakeDb(object):
```
Comment (hamir) [inline fake_db.py]: can you leave FakeDb as is?

File: Procfile [hunk 2]
```diff
@@ -1 +1 @@
-web: python app.py
\ No newline at end of file
+web: FLASK_APP=main.py flask run
```
Comment (hamir) [inline Procfile]: ?

File: app.py [hunk 3]
```diff
@@ -1,27 +1,22 @@
-#!/usr/bin/env python
 import os
 import json
-import util
 
-from flask import Flask, request, make_response, render_template, redirect
-from flask_assistant import Assistant, tell
```
Comment (hamir) [inline app.py]: why remove these deps?

File: app.py [hunk 4]
```diff
@@ -92,16 +87,30 @@ def authenticate():
 
   return redirect(full_redirect_uri)
 
+# /insert?userId=userId1&value=test_value234
+@app.route('/insert', methods=['GET'])
```
Comment (hamir) [inline app.py]: probably shouldn't open these methods up otherwise anyone can mess with our data

File: app.py [hunk 5]
```diff
@@ -92,16 +87,30 @@ def authenticate():
...
+  user_id = request.args.get('userId')
+  _db.DeleteUser(user_id)
+  return 'deleted'
+
 @app.route('/privacy', methods=['GET'])
 def render_privacy():
   return render_template('privacy.html')
 
 
-if __name__ == '__main__':
```
Comment (hamir) [inline app.py]: leave as is?

File: start_time_intent_handler.py [hunk 6]
```diff
@@ -77,11 +77,11 @@ def HandleIntent(self, device_params, post_params):
       # at this stage, we don't know anything about the user's location - try
       # checking our db for a stored location
       user_info = self.fake_db_.GetUserInfo(user_id)
-      if user_info and user_info.get('lat') and user_info.get('lng') and user_info.get('city'):
-        print 'found user ', user_id, ' in databse, so location request is not necesary'
-        lat = user_info.get('lat')
-        lng = user_info.get('lng')
-        city = user_info.get('city')
+      lat = user_info.get('lat')
```
Comment (hamir) [inline start_time_intent_handler.py]: what if user_info is None?

File: requirements.txt [hunk 7]
```diff
@@ -1,4 +1,3 @@
-Flask==0.10.1
+Flask==0.12.2
 requests==2.13.0
-flask-assistant==0.2.5
```
Comment (hamir) [inline requirements.txt]: might need this?

File: main.py [hunk 8]
```diff
@@ -0,0 +1,5 @@
+from flask import Flask
```
Comment (hamir) [inline main.py]: not understanding the benefit of moving from app to main

**Comment by mutahirqureshi** [comment]
looks like sqlite will not work on Gcloud, Gcloud doesn't let you write on the local filesystem. Might have to use Google Cloud Datastore

**Pull request closed without merging**